.env

```
@@ -0,0 +1,10 @@
+API_HOST=127.0.0.1 
+DATABASE_URL=postgres://docker:docker@localhost:5431/postgres 
+DEBUG=
+PORT=80 
+REDIS_HOST="redis:6378" 
+LOKI_HOST=loki
+LOKI_PORT=3101
+LOKI_WRITE_PCT=100 
+SUPERUSER_EMAIL=admin@pinejs.com
```

It'd be better to use `example.com` as it's specifically reserved for this sort of example usage
```suggestion
SUPERUSER_EMAIL=admin@example.com
```